circle.yml

```diff
@@ -1,11 +1,16 @@
 version: 2.1
+
+orbs:
+  aws-s3: circleci/aws-s3@1.0.11
+
 jobs:
-  build-frontend:
+  test-frontend:
```

Comment:
I see a couple console errors in the [frontend tests](https://circleci.com/gh/open-craft/opencraft/9606l). Should we create a follow-up to fix them?

Reply:
They were merged like that because the JS tests/checks were not properly set up. I will create a follow up task to address those issues.